web/js-src/polyfills.js

```diff
@@ -26,6 +26,11 @@ ruffle_script_src+="ruffle.js";
  * keep native objects out of the DOM, and thus out of JavaScript's grubby
  * little hands, but only if we load first.
  */
+
+function insert_after(original_element, new_element) {
+    original_element.parentElement.insertBefore(new_element, original_element.nextSibling);
+}
```

Comment:
Maybe `original_element.insertAdjacentElement("afterend", new_element);`, and then it doesn't have to be a separate function?